### ISSUE TITLE
Fix out of space error on Jenkins build.

### ISSUE THREAD

**Issue by tobeycarman** [ua-snap/dvm-dos-tem#263]
Looks like it only gets thru year 350 of the eq run.

Maybe we need to reduce the default data output on Jenkins? Maybe we need to reduce both the calibration data that is produced and the netcdf stuff?

**Comment by tobeycarman** [comment]
Yikes, the NetCDF outputs are voluminous!

With default settings, this short run:

`$ ./dvmdostem -l info -c -p 2 -e 5 -s 7 -t 4 -n 4 --inter-stage-pause --tar-caljson`

Produces the following:

netcdf outputs:
```
$ du -h DATA/SewardPen_10x10/output/
 89M	DATA/SewardPen_10x10/output/
```

Archived calibration json outputs:
```
$ du -h /tmp/*-data.tar.gz
272K	/tmp/eq-data.tar.gz
 48K	/tmp/pr-data.tar.gz
212K	/tmp/sc-data.tar.gz
372K	/tmp/sp-data.tar.gz
216K	/tmp/tr-data.tar.gz
```

Raw calibration json outputs (last stage only!!):
```
$ du -h /tmp/dvmdostem/
144K	/tmp/dvmdostem//calibration/daily
876K	/tmp/dvmdostem//calibration/monthly
 96K	/tmp/dvmdostem//calibration/yearly
1.1M	/tmp/dvmdostem//calibration
1.1M	/tmp/dvmdostem/
```

**Comment by rarutter** [comment]
Lovely. I'll test with the lowest possible settings, and probably disable
the NetCDF output by default.

**Comment by tobeycarman** [comment]
yeah, or maybe leave a minimal subset of netcdf output working, because then it gets tested each (at least a little bit) every time Jenkins builds.

And / or we could amend the Jenkins builds/test runs so it doesn't do all 1000 years or eq stage.

One of my original ideas was to have the Jenkins builds go all the way thru the process of creating a set of output plots for a "base" case run, which is why I think we set it to do all 1000 years of eq stage. I think the plots are getting generated too, and stored on the Jenkins server, but I never found a good way to make them browsable for everyone else, so the feature is basically un-used.

So maybe we just make the Jenkins build run a much smaller time series, and produce a limited set of outputs?

**Comment by rarutter** [comment]
Some of the outputs were set to daily still, which shouldn't have happened. I did include the ability to disable NetCDF output per stage, so we can both disable eq output and reduce what Jenkins runs (if it's not useful to run for 1000 years, then probably not worth the time).

I'm doing a test run:
`./dvmdostem -l fatal -p 10 -e 1000 -s 100 -t 100 -n 100`
to verify what is produced by the new default settings. I'll update this comment with directory size when it has completed.

**Comment by rarutter** [comment]
PR #264 should have resolved this (the Jenkins build succeeded), but I have not verified it directly in Jenkins.

**Comment by rarutter** [comment]
Fixed by PR #264
